fe/src/project/AspNetCore/shared/components/DownloadCenterFiles/DownloadCenterFiles.js

```diff
@@ -86,19 +101,47 @@ const DownloadCenterFiles = (props) => {
     const handleChangePage = (event, newPage) => {
         const startDisplayFiles = newPage * FilesConstants.defaultPageSize();
         setPage(newPage)
-        setFiles(props.files.slice(startDisplayFiles, startDisplayFiles + FilesConstants.defaultPageSize()))
+
+        let filesToSearch = props.files
```

Comment:
**suggestion:** such assignments make no sense when it's used only once. Please use props.files directly.